ludwig/features/feature_utils.py

```diff
@@ -51,3 +55,32 @@ def set_str_to_idx(set_string, feature_dict, tokenizer_name):
 def compute_feature_hash(feature: dict) -> str:
     preproc_hash = hash_dict(feature.get(PREPROCESSING, {}))
     return feature[NAME] + "_" + preproc_hash.decode('ascii')
+
+
+def locate_feature_in_list(feature_list: list, feature_name: str) -> Union[dict, None]:
+    # searchs a list of features for the feature with the specified name
+    # returns either the requested feature dictionary or None
+    for f in feature_list:
+        if f[NAME] == feature_name:
+            # return the requested feature dictionary
+            return f
+
+    # requested feature not found in the list
+    return None
+
+def retrieve_feature_hash(
```

Comment:
This should not be needed anymore right?

Reply:
removed function